web/src/settings_config.ts

```diff
@@ -4,6 +4,7 @@ import {$t, $t_html} from "./i18n";
 import {page_params} from "./page_params";
 import type {RealmDefaultSettings} from "./realm_user_settings_defaults";
 import type {StreamSpecificNotificationSettings} from "./sub_store";
+import {StreamPostPolicy} from "./sub_store";
```

Comment:
`StreamPostPolicy` probably could be moved to this module if this import turns out to be bad, but I guess `sub_store` doesn't import anything, so seems fine.